src/vsphere_cpi/lib/cloud/vsphere/vm_config.rb

```diff
@@ -125,9 +125,10 @@ def has_custom_cluster_properties?
     
     def find_clusters(clusters_spec)
       clusters = []
+      datacenter_name = vm_type.datacenter.name
```

Comment:
Can we make this entire method just use `map` instead of this whole non-Rubyish pattern?

Reply:
Done.